xtask/src/tools/npm.rs

```diff
@@ -146,6 +160,51 @@ impl NpmRunner {
         }
     }
 
+    fn set_supergraph_yaml_files_correctly(&self, json_key: &str) -> Result<()> {
+        let fed_version = self.get_federation_version(json_key)?;
+        info!("Running tests with Federation version {}", fed_version);
+        self.update_supergraph_yaml_files(fed_version)?;
+        Ok(())
+    }
+
+    fn update_supergraph_yaml_files(&self, fed_version: String) -> Result<()> {
+        let mut supergraphs_path = self.flyby_directory.clone();
+        supergraphs_path.push("supergraphs");
+        for dir_entry in fs::read_dir(supergraphs_path)? {
+            let path = dir_entry.unwrap().path();
+            if path.extension().unwrap() == "yaml" {
```

Comment:
__Nit__: Should we also handle `.yml` and case variations?

Reply:
Yes indeed, we run it through lowercase and then check with a vector so we should be grand